packages/react/src/components/accordion/Accordion.test.tsx

```diff
@@ -14,4 +15,13 @@ describe('<Accordion /> spec', () => {
     const results = await axe(container);
     expect(results).toHaveNoViolations();
   });
+  it('should open the accordion when accordion header is clicked', async () => {
```

Comment:
I would add a line break between test cases for readability

Reply:
- [ ] add a line break between test cases for readability